samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -2014,405 +2011,59 @@ static void hubbleCreateDatabase(DatabaseAdminClient dbAdminClient,
-	    for (int i = 0; i < mutationsPerTransaction; ++i) {
-		String uuid = keys.get(sid).get(random.nextInt(keys.get(sid).size()));
-		keySetBuilder.addKey(Key.of(sid, uuid));
-	    }
-	    if (isStrong) {
-		ResultSet throwaway = dbClient
-		    .singleUse()
-		    .read("Message",
-			  keySetBuilder.build(),
-			  Arrays.asList("body"));
-		while (throwaway.next()) {}
-	    } else {
-		ResultSet throwaway = dbClient
-		    .singleUse(TimestampBound.ofMaxStaleness(15, TimeUnit.SECONDS))
-		    .read("Message",
-			  keySetBuilder.build(),
-			  Arrays.asList("body"));
-		while (throwaway.next()) {}
-	    }
-	}
-    }
-
-    static void hubbleUpdatesAndReads(DatabaseClient dbClient,
-				      int numMailboxes,
-				      int mutationsPerTransaction,
-				      int numRows,
-				      int numMinutes,
-				      boolean isStrong) {
-	List<List<String>> keys = new ArrayList<>(numMailboxes);
-	for (int i = 0; i < numMailboxes; ++i) {
-	    keys.add(new ArrayList<>(numRows / numMailboxes));
-	}
-
-
-	for (int sid = 0; sid < numMailboxes; ++sid) {
-	    for (int i = 0; i < numRows / numMailboxes / mutationsPerTransaction; ++i) {
-		List<Mutation> mutations = new ArrayList<>();
-		for (int j = 0; j < mutationsPerTransaction; ++j) {
-		    String uuid = UUID.randomUUID().toString();
-		    mutations.add(Mutation.newInsertBuilder("Message")
-				  .set("sid").to(sid)
-				  .set("msg_id").to(uuid)
-				  .set("body").to(loremIpsum)
-				  .build());
-		    keys.get(sid).add(uuid);
-		}
-		dbClient.write(mutations);
-	    }
-	}
-
-	System.out.println("Beginning workload.");
-	int numProcessors = Runtime.getRuntime().availableProcessors() / 2;
-	ExecutorService writeExecutor = Executors.newFixedThreadPool(numProcessors);
-	ExecutorService readExecutor = Executors.newFixedThreadPool(numProcessors);
-
-	for (int i = 0; i < numProcessors; ++i) {
-	    writeExecutor.submit(new Runnable() {
-		    @Override
-		    public void run() {
-			hubbleUpdates(dbClient,
-				      keys,
-				      mutationsPerTransaction,
-				      numMinutes);
-		    }
-		});
-
-	    readExecutor.submit(new Runnable() {
-		    @Override
-		    public void run() {
-			hubbleReads(dbClient,
-				    keys,
-				    mutationsPerTransaction,
-				    numMinutes,
-				    isStrong);
-		    }
-		});
-	}
-
-	writeExecutor.shutdown();
-	readExecutor.shutdown();
-	while (!writeExecutor.isTerminated()) {}
-	while (!readExecutor.isTerminated()) {}
-    }
-
   static void run(
       DatabaseClient dbClient,
       DatabaseAdminClient dbAdminClient,
       InstanceAdminClient instanceAdminClient,
       String command,
       DatabaseId database,
       BackupId backup) {
+      CodeLab1 codeLab1=new CodeLab1();
     switch (command) {
       case "createdatabase":
         createDatabase(dbAdminClient, database);
         break;
       case "hubbleCreateMessages":
-	hubbleCreateMessages(dbAdminClient, database);
+          codeLab1.hubbleCreateMessages(dbAdminClient, database);
```

Comment:
Let's remove the `hubble` prefix from all of these functions, now that they are in a separate file. They were there to differentiate these functions from the preexisting samples.